### DISCUSSION TITLE
mylast version97b

### DISCUSSION THREAD

**Discussion by trei49** [KeyPuncher/WiinUPro#186] [Help]
irecentlywashospitalizebecuzmajor strokemylefthand now isgone... and last yr oct a long time unable to use mypc n controller months but i got limited access now but i discovered thrcontroller caannotsuddenly cannot connect anymore idontknow why.i were able to use in may for a bit a while thoughworkfine then wtf this past2month?idungeddit?
i dl your nwesr 98 version i tried to use the setup file but half way through the install i t says about x360 driverissues?
help me pls i am desperate i i dont have other controllers this one i  love very much and a bit impairedhave70%of right brain left due to the stroke i do not  i donot know wat do do how to troubleshoot this by myself
 dont rember how i get this work at all originally

**Comment by KeyPuncher** [comment]
You likely do not need to install the xbox 360 drivers unless you are on Windows 7 or below and haven't had WiinUPro installed on the computer you are using before, both would have to be true, so can skip that.
If you need to/do install the SCP drivers (which are old but still work) uncheck the Bluetooth option.

If the controller isn't connecting anymore, check to make sure any controllers Windows says are paired/remembered are removed and try again.
